plugins/scanners/dos/src/main/kotlin/DOS.kt

```diff
@@ -235,6 +236,7 @@ class DOS internal constructor(
                     }
                     "failed" -> {
                         logger.error { "Scan failed" }
+                        issues += createAndLogIssue(name, "Scan failed in DOS API")
```

Comment:
As `createAndLogIssue()` also logs, I believe we should drop the logger line one above.